examples/testresample.rs

```diff
@@ -12,27 +12,30 @@ fn main() {
     let mut off = 0;
     let mut avail = INBLOCK as isize;
 
-    let fin : Vec<f32> = (0 .. INBLOCK * 2).map(|i| ((i as f32) / PERIOD * 2.0 * PI).sin() * 0.9).collect();
+    let fin: Vec<f32> = (0..INBLOCK * 2)
+        .map(|i| ((i as f32) / PERIOD * 2.0 * PI).sin() * 0.9)
+        .collect();
```

Comment:
this is `rustfmt` at work, hope you don't mind

Reply:
I do not, but formatting and functional changes should live in different commits.